webpack/webpack.common.js

```diff
@@ -46,19 +46,6 @@ module.exports = {
             },
           },
         ],
-      },
```

Comment:
ok, now I see you are using a different method

Reply:
?